dht.go

```diff
@@ -312,19 +312,18 @@ func (dht *IpfsDHT) betterPeersToQuery(pmes *pb.Message, p peer.ID, count int) [
 
 	// no node? nil
 	if closer == nil {
+		log.Warning("no closer peers to send:", p)
```

Comment:
No, this can only happen if the routing table is empty, and i think having an empty routing table while trying to respond to a peer who definitely has you in their routing table might be worthy of a warning log